styles/style.css

```diff
@@ -0,0 +1,202 @@
+nav{
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0rem 11.6rem;
+}
+nav ul{
+    list-style-type: none;
+    margin-right: 1rem;
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+}
+nav ul li{
+    display: inline-block;
+    margin-right: 1.4rem;
+    text-align: center;
+}
+nav ul li a{
+    text-decoration: none;
+    font-weight: 700;
+    color: grey;
+    text-align: center;
+}
+.setloc{
+    color: black;
+}
+.swiggy-logo{
+    width: 50px;
```

Comment:
Always use max-width to give dimensions to images. Giving fixed width or height will result in pixelation if the image resolution or the image itself changes in the future